### PR TITLE
Designer tools

### PR DESCRIPTION
**Pull request by mthjn**
Added some webdesign resources, quite edged around Google Material Design
https://github.com/mthjn/free-for-dev#design--ui

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -347,7 +349,17 @@ Table of Contents
...
+     
+## Design & UI  
+  
+* [Codepen - Snippets](http://materialdesignblog.com/top-25-material-design-html-css-code-snippets-from-codepen/) - Top 25 Codepen Snippets for Material Design
+* [UI8](https://ui8.net/category/freebies) - Freebies at UI8 (mockups, icons, themes)
+* [Sketch](http://www.sketchappsources.com/all-free-sources.html) - Free design resources for SKETCH (but sometimes plain SVG too)
+* [Graphs Collection](https://www.materialup.com/collections/graphs-87b50f17-370d-4826-b9c1-cb072badd468) - Material Design graphs collection
+* [Materializer](https://gitlab.datatrends.es/opensource/materializer) - The most complete Material Design Framework
+* [Pixlr](http://pixlr.com) - Free online browser editor  
+* [CreativeCloudTrial](http://www.adobe.com/creativecloud/catalog/desktop.html?promoid=KSPAJ) - Free 30 day Trial on Adobe creative cloud (Illustrator, Photoshop, After Effects ...) You get 30 days of a program being installed, if you uninstall the program after having used it the countdown pauses.
```
Comment (ripienaar) [inline README.md]: Hello,

Appreciate you adding these, but this list is for software as a service offerings with free tiers.  Most of these here does not match the stated goal of the list.

Mind reviewing and remove the ones that aren't *aaS? This excludes downloads, documentation, guides, trials etc

thanks

Comment (mthjn) [inline README.md]: Hi, I see. I have removed all non SaaS and added , guess I'll make a separate list for them. Please check the [latest commit](https://github.com/mthjn/free-for-dev/commit/df41fb9d610769077990420f1b17826ee3f616e6).

**Comment by ripienaar** [comment]
nice, thanks for this - looks like it wont merge cleanly though :( can you rebase to master?

**Comment by mthjn** [comment]
I suspect the problem was I didn't have the latest version of your repo, I have updated mine. Also cleaned up the other branches that weren't needed anyway.

**Comment by ripienaar** [comment]
erm, wow this has gone a bit crazy now its changing tons of things lol :) def looks like you're on quite a old version, def cant merge as it is now

**Pull request closed without merging**